### PR TITLE
[+] Types Support - Support Point field type

### PR DESCRIPTION
**Pull request by lucasscariot**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request
- [x] Write changes made in the CHANGELOG.md
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by lucasscariot** [comment]
[(5) Bon Debarras wants the Express-Sequelize liana to support Points/Geometry](https://trello.com/c/uRsFY150/1931-5-bon-debarras-wants-the-express-sequelize-liana-to-support-points-geometry)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Record Serialization - Type Point are serialized and in ressources attibutes.
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: `Types Support - Support Point field type.`

File: deserializers/resource.js [hunk 2]
```diff
@@ -20,6 +20,16 @@ function ResourceDeserializer(Implementation, model, params,
...
 
+    _.each(schema.fields, function (field) {
+      if (field.type === 'Point' && params.data.attributes[field.field]) {
+        var value = params.data.attributes[field.field];
+        params.data.attributes[field.field] = {
+          type: 'Point',
+          coordinates: value.split(',')
+        };
+      }
+    });
```
Comment (arnaudbesnier) [inline deserializers/resource.js]:
```javascript
var coordinates = params.data.attributes[field.field].split(',');
params.data.attributes[field.field] = {
  type: 'Point',
  coordinates: coordinates
};
```
is easier to read I think

File: serializers/resource.js [hunk 3]
```diff
@@ -17,6 +17,7 @@ function ResourceSerializer(Implementation, model, records, integrator,
 
   var reservedWords = ['meta'];
   var fieldNamesDateonly = [];
+  var fieldPoints = [];
```
Comment (arnaudbesnier) [inline serializers/resource.js]: `fieldNamesPoint` this is the array of field name having the type `Point`.

**Pull request merged**